test/Logical/TransformationsTest.jl

```diff
@@ -148,6 +148,22 @@ end
     @test n1.children[1] == n2.children[1]
 end
 
+"""
+This reproduces the bug that for multiple depths, the result is the same
+"""
```

Comment:
I think we cannot add tripple quote comments before "testtset" here. Error from the travis log:

```
Error During Test at /home/travis/build/Juice-jl/LogicCircuits.jl/test/Logical/TransformationsTest.jl:151
  Got exception outside of a @test
  LoadError: cannot document the following expression:
.
.
.
.

 '@testset' not documentable. See 'Base.@__doc__' docs for details.
```

Reply:
Updated in latest commit.